client/config.go

```diff
@@ -3,12 +3,12 @@ package client
 // Provider Configuration
 
 type Config struct {
-	Subscriptions []string `yaml:"subscriptions"`
+	Subscriptions []string `hcl:"subscriptions"`
 }
 
 func (c Config) Example() string {
 	return `configuration {
 		//  Optional. if you not specified, cloudquery tries to access all subscriptions available to tenant
-		subscriptions = ["<YOU_SUBSCRIPTION_ID_HERE">]
```

Comment:
YOUR (and in three other places)

Reply:
this is a removed line
my line looks next `subscriptions = ["<YOU_SUBSCRIPTION_ID_HERE>"]`

Reply:
this is a removed line
my line looks next: `subscriptions = ["<YOU_SUBSCRIPTION_ID_HERE>"]`

Reply:
I know, you just swapped " and > characters. I mean while we are at it, we could also fix spelling. The intent here is "youR subscription id".